grails-app/controllers/org/broadinstitute/orsp/SubmissionController.groovy

```diff
@@ -7,6 +7,20 @@ import org.broadinstitute.orsp.utils.IssueUtils
 @Slf4j
 class SubmissionController extends AuthenticatedController {
 
+    def show() {
+        Issue issue = queryService.findByKey(params.id)
+        if (issueIsForbidden(issue)) {
+            redirect(controller: 'Index', action: 'index')
```

Comment:
Is there no user message when this happens? This seems like a poor user experience to follow a link only to not go where you tried to with no explanation as to why. I suggest adding a story for this as it really is unrelated to the larger story of integrating the various react component pages.